source/shared/core_stmt.cpp

```diff
@@ -2637,28 +2591,6 @@ void resize_output_buffer_if_necessary( _Inout_ sqlsrv_stmt* stmt, _Inout_ zval*
     }
 }
 
-bool reset_ae_stream_cursor( _Inout_ sqlsrv_stmt* stmt ) {
-    // only handled differently when AE is on because AE does not support streaming
-    // AE only works with SQL_CURSOR_FORWARD_ONLY for max types
-    if (stmt->conn->ce_option.enabled == true && stmt->current_results->odbc->cursor_type == SQL_CURSOR_FORWARD_ONLY) {
-        // close and reopen the cursor
-        core::SQLCloseCursor(stmt->current_results->odbc);
-        core::SQLExecute(stmt);
-
-        // advance to the previous active result set
-        for (int j = 0; j < stmt->curr_result_set; j++) {
-            core::SQLMoreResults(stmt);
-        }
-
-        // FETCH_NEXT until the cursor reaches the row that it was at
-        for (int i = 0; i <= stmt->fwd_row_index; i++) {
-            core::SQLFetchScroll(stmt->current_results->odbc, SQL_FETCH_NEXT, 0);
-        }
-        return true;
-    }
-    return false;
-}
-
 void adjustInputPrecision( _Inout_ zval* param_z, _In_ SQLSMALLINT decimal_digits ) {
     size_t maxDecimalPrecision = 38;
```

Comment:
why magic number 38?

Reply:
@lilgreenbird I think she has a comment about this magic number a few lines below.